web/packages/teleport/src/DesktopSession/useTdpClientCanvas.tsx

```diff
@@ -126,7 +126,6 @@ export default function useTdpClientCanvas(props: Props) {
   ) => {
     // The first image fragment we see signals a successful TDP connection.
     if (!initialTdpConnectionSucceeded.current) {
-      syncCanvas(ctx.canvas, getDisplaySize());
```

Comment:
This was not needed here in the first place (likely vestigial from a prev iteration), and was getting in the way of desired `screen_size` behavior.